app/controllers/ops_controller/settings/common.rb

```diff
@@ -448,7 +448,15 @@ def settings_update_save
       @changed = (@edit[:new] != @edit[:current].config)
       @update = VMDB::Config.new("vmdb")                    # Get the settings object to update it
     when "settings_advanced" # Advanced manual yaml editor tab
-      save_advanced_settings(MiqServer.find(@sb[:selected_server_id]))
+      nodes = x_node.downcase.split("-")
+      resource = if selected?(x_node, "z")
+                   Zone.find(nodes.last)
+                 elsif selected?(x_node, "svr")
+                   MiqServer.find(@sb[:selected_server_id])
+                 else
+                   MiqRegion.my_region
+                 end
```

Comment:
Also minor, but this is also a duplicated pattern with the selected_node_title methods.  I think this would be cleaner extracted as a `selected_node` method that lives with the rest in ops_helper (if possible).  In this way all of the "if selected?(x_node..." logic lives in one place.